components/whyd/2022/Terminal.vue

```diff
@@ -6,13 +6,15 @@
         @clicked="terminalMouseDown"
       />
       <div id="textViewport" class="center" @click="focusInput">
-        <p
-          v-for="(line, index) in displayedTerminalLines"
-          :key="index"
-          :class="line.class"
-        >
-          {{ line.content }}
-        </p>
+        <div v-for="(text, index) in displayedTerminalLines" :key="index">
```

Comment:
`displayedTerminalLines` probably needs a new name since you're pulling something more complicated than `lines` from it